Competition2017/src/competition/subsystems/collector/commands/EjectCollectorCommand.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.collector.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.collector.CollectorSubsystem;
+import xbot.common.command.BaseCommand;
+
+
+public class EjectCollectorCommand extends BaseCommand {
+    final CollectorSubsystem collectorSystem;
+    
+    @Inject
+    public EjectCollectorCommand(CollectorSubsystem collectorSystem){
+        this.collectorSystem = collectorSystem; 
+        this.requires(collectorSystem);
+    }
+    @Override
+    public void initialize() {
+        
```

Comment:
Create a logger for this class in the constructor, and use it here:

"initializing EjectCollectorCommand"